src/Components/Item/Item.module.css

```diff
@@ -5,3 +5,14 @@
 .done {
   color: #333;
 }
+
+.list {
+  list-style-type: none;
+  text-align: left;
+  padding: 0;
+}
+
+.delete {
+  position: relative;
+  float: right;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Olga_project_React/pullRequest?prid=6231802)